typer/main.py

```diff
@@ -356,7 +356,7 @@ def get_group_from_info(group_info: TyperInfo) -> click.Command:
         convertors,
         context_param_name,
     ) = get_params_convertors_ctx_param_name_from_function(solved_info.callback)
-    cls = solved_info.cls or click.Group
```

Comment:
So from the Click docs `cls` is the `the argument class to instantiate` (which, from my understanding, that could be a Command or a Group). I believe this change will basically create groups of single commands instead of creating a group or a command depending on what you need. If you need just a command, then it may not make sense to create a group (extra information in memory), but I'm not too sure.

Reply:
I'm not sure of this one (don't know enough about the Click internals).